### PR TITLE
Stop program crash when modifying book not found in database

### PR DESCRIPTION
**Pull request by Nelwell**
@KhadarRashid Look over changes made to stop program from crashing when changing book-read status in bookstore.py and the check added in main.py for prevent modifying a book that's not in the db to begin with. #5

### CODE REVIEW

**Comment by KhadarRashid** [comment]
Good job with adding that if else statement in main.py looks like a good way to solve the issue.  But what if the user doesn't want to stay stuck in the change_read(). Could you possibly add a way for them to go back to the main menu as well?

**Comment by Nelwell** [comment]
ok I made the suggested addition. Thanks for the feedback.  Check it out when you can.

**Comment by KhadarRashid** [comment]
This looks good to me. Just what I imagined.

File: main.py [hunk 1]
```diff
@@ -66,7 +66,11 @@ def change_read():
         book.save()
     else:  # If book ID not found in db
         print('That book is not found.')
-        change_read()  # restarts function for book ID input
+        option = input('Return to main menu? Y for main menu. ').upper()
+        if option == 'Y':
+            print()
+        else:
+            change_read()  # restarts function for book ID input
```
Comment (claraj) [inline main.py:73]: If the function ends you'll be taken back to the main menu, this is probably more complex than needed. Also use the ui methods to get input and display messages

**Pull request merged**